cmd/keeper/keeper.go

```diff
@@ -93,6 +95,8 @@ func init() {
 	cmdKeeper.PersistentFlags().StringVar(&cfg.pgConfDir, "pg-conf-dir", "", "absolute path to user provided postgres configuration. If empty a default dir under $dataDir/postgres/conf.d will be used")
 	cmdKeeper.PersistentFlags().StringVar(&cfg.pgSUUsername, "pg-su-username", "", "postgres superuser user name (required by pg_rewind)")
 	cmdKeeper.PersistentFlags().StringVar(&cfg.pgSUPassword, "pg-su-password", "", "postgres superuser password (required by pg_rewind)")
+	cmdKeeper.PersistentFlags().StringVar(&cfg.initialSUUsername, "initial-su-username", "", "postgres initial superuser username")
+	cmdKeeper.PersistentFlags().StringVar(&cfg.initialSUPasswordFile, "initial-su-password-secret-file", "", "postgres initial superuser password secret file")
```

Comment:
To make it consistent I'd like to leave the pg prefix. But the options are becoming quite long :stuck_out_tongue_closed_eyes: 

what about:

`initial-pg-su-username`
`initial-pg-su-password-file`
?